<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItem</class>
 <widget class="QDialog" name="AddItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить в наряд...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Основная информация об имуществе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Номенклатурный номер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNomNumb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Категория</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCategory">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Единица измерения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMeasure">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Введите данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Количество*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="SpinBoxCount">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <double>9999999.990000000223517</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Заводские номера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serialNumb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Введите через запятую (1040802213018, 1040802614013..)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Дата выпуска</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="date">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Введите через запятую (25.10.1997, 11.11.2011..)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(160, 10, 10);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 160,10);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
